### ISSUE TITLE
Plugin cannot find assets from .css file

### ISSUE THREAD

**Issue by marcin-piela-sf** [happo/happo-cypress#27]
We have fonts.css file with relative paths to font file and plugin cannot send it. It tries to fetch it with following url

```
[HAPPO] Failed to fetch url http://localhost:3000/../fonts/font1.woff — Not Found
```
instead of
```
http://localhost:3000/assets/fonts/font1.woff
```

fonts.css file example:
```
@font-face {
  font-family: "font1";
  src: url("../fonts/font1.woff2") format("woff2"),
    url("../fonts/font1.woff") format("woff");
}
```

Versions:
```
"happo-cypress": "^1.7.2",
"happo.io": "^5.6.1",
```

**Comment by trotzig** [comment]
Thanks for the report! I'll have a closer look at this soon.

**Comment by trotzig** [comment]
A fix for this was included in version 1.7.3 (that I just released). Again, thanks for the report!
